vulnerabilities/package_managers.py

```diff
@@ -97,7 +122,16 @@ async def fetch(self, pkg, session):
         try:
             response = await session.request(method="GET", url=url)
             response = await response.json()
-            versions = set(response["releases"])
+            for version in response["releases"]:
```

Comment:
What about this to avoid iterating twice:
```
for version, released_downloads in response["releases"].items():
    if not released_downloads:
        continue
    version.add(....

```

otherwise you are making three times a lookup in `response["releases"][version]`